src/HearThis/UI/ToastNotifier.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
```

Comment:
Seems like this should be part of libpalaso, rather than having a separate copy in both HearThis and Bloom.